src/Kiota.Builder/PathSegmenters/JavaPathSegementer.cs

```diff
@@ -7,5 +7,5 @@ public class JavaPathSegmenter : CommonPathSegmenter
     public JavaPathSegmenter(string rootPath, string clientNamespaceName) : base(rootPath, clientNamespaceName) { }
     public override string FileSuffix => ".java";
     public override string NormalizeFileName(CodeElement currentElement) => GetLastFileNameSegment(currentElement).ToFirstCharacterUpperCase();
-    public override string NormalizeNamespaceSegment(string segmentName) => segmentName.ToLowerInvariant();
+    public override string NormalizeNamespaceSegment(string segmentName) => segmentName?.ToLowerInvariant() ?? string.Empty;
```

Comment:
Similar comment here. Doesn't NRT give us the assurance that `segmentName` is not null?

Reply:
that's what I thought as well but apparently if a consumer of the lib doesn't have NRT enabled, they'll be able to pass null values. I'm guessing they did that for compatibility reasons.

Reply:
Oh. I see. 
Thanks for clearing that one up.